cuckoo/cuckoo_main.py

```diff
@@ -1084,6 +1084,9 @@ def _remove_illegal_characters_from_file_name(self) -> None:
             self.log.debug(f"Renaming {self.file_name} because it contains one of {ILLEGAL_FILENAME_CHARS}")
             self.file_name = ''.join(ch for ch in self.file_name if ch not in ILLEGAL_FILENAME_CHARS)
 
+        if self.file_name.startswith(" "):
+            self.file_name = self.file_name.lstrip()
```

Comment:
Does the bug also happen with trailing whitespace?  What about other whitespace characters (tab etc)? Is there any reason not to use `file_name.strip()` without a check?

Reply:
It only happens with leading whitespace